media-source-respec.html

```diff
@@ -418,73 +422,158 @@ <h3>Attributes</h3>
               </ol>
             </li>
 
-            <li>Let <var>SourceBuffer audioTracks list</var> equal the {{AudioTrackList}} object returned by <var>sourceBuffer</var>.{{SourceBuffer/audioTracks}}.</li>
-            <li>If the <var>SourceBuffer audioTracks list</var> is not empty, then run the following steps:
+            <li>Let |SourceBuffer audioTracks list| equal the {{AudioTrackList}} object returned by
```

Comment:
It is a good idea to use the `|` shorthand syntax instead of `<var>`. This makes ReSpec track and highlight variables when user hovers or clicks on them as in https://user-images.githubusercontent.com/8426945/74150174-5adf9f00-4c2f-11ea-82ee-d7c059a8f772.gif

That may exceed the scope of this PR but I note that it would be equally good to replace constructs such as `<var>sourceBuffer</var>` with `|sourceBuffer|` throughout algorithms as well.

Reply:
I agree - I used `|` in much of the PR. I'll fix portions in this PR that touch lines with `<var>`, too. There are currently over 300 uses, so fixing the remaining bulk will be a different PR.

Reply:
cf47a8f addresses this feedback. There still remain over 200 occurrences in the spec after that.